charactersheet/charactersheet/templates/character/traits.tmpl.html

```diff
@@ -243,6 +261,16 @@ <h4 class="modal-title">Edit a Trait</h4>
                 </div>
               </div>
               <div class="form-group">
+                <label for="race" class="col-sm-2 control-label">Race</label>
+                <div class="col-sm-10">
+                  <input type="text"
+                         class="form-control"
+                         id="race"
+                         placeholder="Trance"
+                         data-bind='textInput: race, hasFocus: $parent.editFirstModalElementHasFocus'>
```

Comment:
This field shouldn't have focus. `name` should be the only one that has this property.